docs/index.rst

```diff
@@ -204,8 +204,6 @@ These are recipes that'll help you address specific use-cases.
 
    how-tos/*
 
-- :doc:`how-tos/casting-type-to-custom-json`
-- :doc:`how-tos/embedding-table-from-another-schema`
 - :doc:`how-tos/providing-images-for-img`
 - `How PostgreSQL triggers work when called with a PostgREST PATCH HTTP request <https://blog.fgribreau.com/2020/11/how-postgresql-triggers-works-when.html>`_
```

Comment:
How about moving this one to the ecosystem page? We can have a `Community How-to's` section there.
```suggestion
```

Reply:
Yep, that's fine as well